bootstrap-salt.sh

```diff
@@ -1440,7 +1440,7 @@ __check_dpkg_architecture() {
             else
                 # Saltstack official repository does not yet have arm64 metadata,
                 # use amd64 repositories on arm64, since all pkgs are arch-independent
```

Comment:
Wouldn't this comment also need to change?

Reply:
Good catch.